### PR TITLE
ENT-3181: Enables custom branding for enterprise customers in logo and menu

### PR DESCRIPTION
**Pull request by jlajoie**

### CODE REVIEW

File: src/course-header/Header.jsx [hunk 1]
```diff
@@ -34,21 +34,46 @@ export default function Header({
...
   const { authenticatedUser } = useContext(AppContext);
 
-  const { enterpriseLearnerPortalLink } = useEnterpriseConfig(
+  const { enterpriseLearnerPortalLink, enterpriseCustomerBrandingConfig } = useEnterpriseConfig(
     authenticatedUser,
     getConfig().ENTERPRISE_LEARNER_PORTAL_HOSTNAME,
     getConfig().LMS_BASE_URL,
   );
 
+  let dashboardMenuItem = <Dropdown.Item href={`${getConfig().LMS_BASE_URL}/dashboard`}>Dashboard</Dropdown.Item>;
```
Comment (ciduarte) [inline src/course-header/Header.jsx]: Should we i18n `Dashboard`?

**Review by ciduarte — APPROVED** [review]
Looks good! 😁 one comment about i18n

File: src/course-header/Header.jsx (hunk 1 above)
Comment (jlajoie) [reply src/course-header/Header.jsx]: Your call, just moved that from elsewhere in this file 😄

Comment (ciduarte) [reply src/course-header/Header.jsx]: Yeah let's throw that into the nearest messages and `intl` that sucker

**Pull request merged**